www-apps/redmine/redmine-3.4.11.ebuild

```diff
@@ -0,0 +1,232 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+USE_RUBY="ruby23 ruby24"
+inherit eutils depend.apache ruby-ng user
+
+DESCRIPTION="Flexible project management web application using the Ruby on Rails framework"
+HOMEPAGE="https://www.redmine.org/"
+SRC_URI="https://www.redmine.org/releases/${P}.tar.gz"
+
+KEYWORDS="~amd64"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="imagemagick fastcgi ldap markdown mysql passenger postgres sqlite"
+
+ruby_add_rdepend "
+	fastcgi? ( dev-ruby/fcgi )
+	imagemagick? ( >=dev-ruby/rmagick-2.16.0:0 )
+	ldap? ( >=dev-ruby/ruby-net-ldap-0.12.0 )
+	markdown? ( >=dev-ruby/redcarpet-3.4.0 )
+	mysql? ( >=dev-ruby/mysql2-0.4.6:* )
+	passenger? ( www-apache/passenger )
+	postgres? ( >=dev-ruby/pg-0.18.1:0 )
+	sqlite? ( >=dev-ruby/sqlite3-1.3.12 )
+	dev-ruby/actionpack-action_caching
+	dev-ruby/actionpack-xml_parser:0
+	>=dev-ruby/builder-3.2.2:3.2
+	>=dev-ruby/coderay-1.1.1
+	dev-ruby/i18n:0.7
+	>=dev-ruby/jquery-rails-3.1.4:3
+	dev-ruby/loofah
+	dev-ruby/mime-types:3
+	dev-ruby/mimemagic
+	>=dev-ruby/nokogiri-1.8.1
+	dev-ruby/protected_attributes
+	>=dev-ruby/rack-openid-0.2.1
+	>=dev-ruby/rails-4.2.11.1:4.2
+	>=dev-ruby/rails-html-sanitizer-1.0.3
+	dev-ruby/request_store:1.0.5
+	>=dev-ruby/roadie-rails-1.1.1
+	>=dev-ruby/rbpdf-1.19.6
+	>=dev-ruby/ruby-openid-2.3.0
+	dev-ruby/rubygems
+	"
+
+REDMINE_DIR="/var/lib/${PN}"
+
+pkg_setup() {
+	enewgroup redmine
```

Comment:
Needs to be handled via the GLEP-81 way, many examples already on gentoo-dev mailing list.